### PR TITLE
Associate taxonomy permissions with Editor role

### PR DESCRIPTION
**Pull request by zackdavies**
Associate view/create/edit permissions for Event taxonomies with Editors. See https://github.com/localgovdrupal/localgov_events/issues/136

### CODE REVIEW

**Comment by zackdavies** [comment]
I've got another PR open for this repo and both include hook_update_X - prior to merging (if approved!) X should be a different number for each.

**Comment by ekes** [comment]
Related https://github.com/localgovdrupal/localgov_topics/pull/28

I'm certain for this one that lots of sites fix the locations and don't let editors change them.

That said. Installing the site with Editors able to set the taxonomy terms and letting people remove change the permission might be a very valid way of doing this.

Not sure about the update hook in that case though. I think plenty of existing sites will have configured their permissions as desired for their use case.

File: localgov_events.install [hunk 1]
```diff
@@ -73,3 +73,34 @@ function localgov_events_update_8002() {
     $config->save(TRUE);
   }
 }
+
+/**
+ * Expanding Events permissions.
+ *
+ * Grant taxonomy term permissions to Editor role.
+ */
+function localgov_events_update_10001($is_syncing) {
```
Comment (ekes) [inline localgov_events.install]: MT: Good to do this as the default for new installs; but not updating existing sites.

**Review by ekes — CHANGES_REQUESTED** [review]
So just removing the update before merging.

File: localgov_events.install (hunk 1 above)
Comment (zackdavies) [reply localgov_events.install]: OK, removed.

**Review by stephen-cox — APPROVED** [review]
Discussing at Merge Tuesday - looks good to add the permissions for these taxonomy terms.

**Pull request merged**